backend/v1/project.py

```diff
@@ -0,0 +1,109 @@
+from flask import Blueprint, jsonify, request
+
+from backend.auth import token_required
+from backend.data import projects
+
+projects_api = Blueprint('ProjectsApi', __name__, url_prefix='/project')
```

Comment:
prefix must be plural '/projects'

Reply:
Then `/user` should also be `/users` I guess.

Reply:
Optional, user would work from client perspective and users would work from admin perspective.